src/devices/tuya.ts

```diff
@@ -10968,6 +10968,50 @@ const definitions: Definition[] = [
             ],
         },
     },
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE204_ncti2pro'}],
+        model: 'TS0601_pn6_6_way_controller',
+        vendor: 'Tuya',
+        description: '6-way controller',
+        whiteLabel: [tuya.whitelabel('ZSVIOT', 'PN6', '6-way controller', ['_TZE204_ncti2pro'])],
```

Comment:
Please remove the whitelabel and update the vendor/model name instead.

Reply:
Done